lib/routes/theinitium/app.ts

```diff
@@ -0,0 +1,136 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    path: '/app/:category?',
+    categories: ['new-media'],
+    example: '/theinitium/app',
+    parameters: {
+        category: 'Category, see below, latest_sc by default',
+    },
+    features: {
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: 'App',
+    maintainers: ['quiniapiezoelectricity'],
+    radar: [
+        {
+            source: ['app.theinitium.com/t/latest/:category'],
+            target: '/app/:category',
+        },
+    ],
+    handler,
+    description: `
+:::warning
+此路由暂不支持登陆认证
+:::
+
+Category 栏目：
+
+| ----- | 简体中文     | 繁體中文      |
+| ----- | ----------------- | ---------------- |
+| 最新   | latest_sc | latest_tc |
+| 日报   | daily_brief_sc | daily_brief_tc |
+| 速递   | whats_new_sc | whats_new_tc |
+| 专题   | report_sc | report_tc |
+| 评论   | opinion_sc | opinion_tc |
+| 国际   | international_sc | international_tc |
+| 大陆   | mainland_sc | mainland_tc |
+| 香港   | hongkong_sc | hongkong_tc |
+| 台湾   | taiwan_sc | taiwan_tc |
+| 播客   | article_audio_sc | article_audio_tc |`,
+};
+
+async function handler(ctx) {
+    const category = ctx.req.param('category') ?? 'latest_sc';
+
+    const feedList = await got({
+        method: 'get',
+        url: `https://app.theinitium.com/timelines.json`,
+    });
```

Comment:
Consider [caching](https://docs.rsshub.app/joinus/advanced/use-cache) the response with `tryGet`. Set 3rd parameter to `config.cache.routeExpire and 4th to `false`